src/components/navigation.rs

```diff
@@ -0,0 +1,18 @@
+use dioxus::prelude::*;
+use dioxus_router::prelude::*;
+use crate::Route;
+
+const NAVBAR_LINK_STYLE: &str = "p-1 px-4 text-center flex items-center text-white hover:border-transparent hover:bg-blue-2 border-2 border-blue-2 rounded-md transition ease-in-out delay-30";
+
+#[allow(non_snake_case)]
+#[inline_props]
+pub fn Navigation(cx: Scope) -> Element {
+    render! {
+        nav { class: "mx-auto p-2 flex justify-stretch bg-blue-1 rounded-md gap-2",
```

Comment:
I think a fully rounded navbar is better 😝

Reply:
I don't know... I kind of like the consistency with the project cards. It also becomes a circle on small screens :/

Reply:
It now uses the fully rounded version on medium screens and wider, but uses the medium version on smaller screens. Trust me, the fully rounded version looks really bad on small screens.